doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -873,6 +873,25 @@ mylibvar = Libdl.dlopen("mylib")
 ccall(@dlsym("myfunc", mylibvar), Void, ())
 ```
 
+## Closing a Library
+
+It is sometimes useful to close (unload) a library so that it can be reloaded.
+For instance, when developing C code for use with Julia, one may need to compile,
+call the C code from Julia, then close the library, make an edit, recompile,
+and load in the new changes. One could either restart Julia, or one can use the
```

Comment:
Perhaps the "could"/"can" in "One could either..." and "... or one can ..." should be consistent?